.github/workflows/stale.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Warn and Close stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '30 5 * * *'
+
+jobs:
+  warn_and_close_stails:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-issue-stale: 90
+        days-before-pr-stale: 45
```

Comment:
Will this GHA add a comment along with a stale label on issues/PRs?

Reply:
Yes check the `stale-pr-message: 'This pull request has not been updated in the past 45 days.'` in this file.